Source/Urho3D/CSharp/Swig/eastl_string.i

```diff
@@ -116,6 +116,7 @@ class string_view;
 // Helpers
 %wrapper %{
     SWIGEXPORT void SWIGSTDCALL CSharp_Urho3D_String_Set(eastl::string* str, const char* val) { *str = val; }
+    SWIGEXPORT const char* SWIGSTDCALL CSharp_Urho3D_String_Get(eastl::string* str) { return str->c_str(); }
```

Comment:
What is that for?

Reply:
I had a linked error a while ago... Probably on iOS, I don't remember now :-(

Reply:
I am suspicious of this. Could you try your CI build with this line commented out so we can find out whats really a problem?